### PR TITLE
Offchain DAOs on Etherlink

### PR DESCRIPTION
**Pull request by ashutoshpw**
- [x] Implementing WAGMI for Etherlink Login
- [x] Moved network definitions to Constant file
- [x] Changing useTezos hook to handle login with Etherlink
- [x] Changing Validation options when Etherlink is active.
- [x] Show logged in user across the website
- [x] Deploy Offchain DAO Form

### CODE REVIEW

File: src/modules/common/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    return (
+      <SmallButton
+        color="secondary"
+        variant="contained"
+        style={{ fontSize: "14px" }}
+        onClick={() => {
+          connect()
+        }}
+      >
+        Connect Wallet C
```
Comment (Man-Jain) [inline src/modules/common/ConnectWalletButton.tsx:29]: Should `C` be here ?

**Review by Man-Jain — CHANGES_REQUESTED** [review]

File: src/modules/common/ConnectWalletButton.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ashutoshpw) [reply src/modules/common/ConnectWalletButton.tsx]: Removed.

File: src/modules/common/ConnectWalletButton.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      </SmallButton>
+    )
+  }
+  return (
+    <SmallButton variant="outlined" onClick={() => connect()}>
+      Connect Wallet A
```
[the remark this answers is not in the document]
Comment (ashutoshpw) [reply src/modules/common/ConnectWalletButton.tsx]: Removed.

File: src/modules/creator/deployment/steps/Ownership.tsx [hunk 3]
```diff
@@ -122,7 +126,17 @@ export const Ownership: React.FC = () => {
                 </OptionButton>
               </Grid>
               <Grid item xs={isMobileSmall ? 12 : 3} container justifyContent={isMobileSmall ? "center" : "flex-start"}>
-                <OptionButton underline="none" href={`/creator/deployment`}>
+                <OptionButton
+                  underline="none"
+                  onClick={() => {
+                    if (etherlink.isConnected) {
+                      window.open(`https://remix.ethereum.org/`)
```
[the remark this answers is not in the document]
Comment (ashutoshpw) [reply src/modules/creator/deployment/steps/Ownership.tsx:133]: We are building a deployer right here:
https://github.com/dOrgTech/homebase-app/pull/868

For the time being, replaced this with : https://www.cookbook.dev/contracts/simple-token

**Comment by Man-Jain** [comment]
@ashutoshpw Are these comment still WIP or done ?

**Comment by ashutoshpw** [comment]
@Man-Jain This is still in WIP, I will re-request review once all the issues as fixed.

**Review by EightRice — APPROVED** [review]

**Review by Man-Jain — APPROVED** [review]

**Pull request merged**